### PR TITLE
Update type test baselines

### PR DESCRIPTION
**Pull request by CraigMacomber**
## Description

```
pnpm exec flub typetests -g client --private --disable
pnpm exec flub typetests -g client --reset --previous --normalize
pnpm i
pnpm run build:fast
```
This disabled type tests in two private packages which did not have type tests, but also didn't mark themselves as having disabled type tests:

-   @fluid-tools/markdown-magic
-   @fluid-private/changelog-generator-wrapper

API breaks in these packages were restored (If the type test baselines were bumped wright after the branch spilt this list should be empty, but 2.0 had a longer fork to release delay than normal for stabilization, as well as a delay before updating these baselines):

-   @fluidframework/odsp-driver-definitions
-   @fluidframework/container-definitions
-   @fluidframework/file-driver
-   @fluidframework/azure-client

## Reviewer Guidance

The review process is outlined on [this wiki page](https://github.com/microsoft/FluidFramework/wiki/PR-Guidelines#guidelines).

### CODE REVIEW

File: packages/common/container-definitions/package.json [hunk 1]
```diff
@@ -100,7 +100,7 @@
 		"@fluid-tools/build-cli": "^0.40.0",
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/build-tools": "^0.40.0",
-		"@fluidframework/container-definitions-previous": "npm:@fluidframework/container-definitions@2.0.0-rc.5.0.0",
+		"@fluidframework/container-definitions-previous": "npm:@fluidframework/container-definitions@2.0.0",
```
Comment (tylerbutler) [inline packages/common/container-definitions/package.json:103]: Noting for follow up - find the commit which would have added the "broken" entries.

File: packages/drivers/odsp-driver-definitions/package.json [hunk 2]
```diff
@@ -95,7 +95,7 @@
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/build-tools": "^0.40.0",
 		"@fluidframework/eslint-config-fluid": "^5.3.0",
-		"@fluidframework/odsp-driver-definitions-previous": "npm:@fluidframework/odsp-driver-definitions@2.0.0-rc.5.0.0",
+		"@fluidframework/odsp-driver-definitions-previous": "npm:@fluidframework/odsp-driver-definitions@2.0.0",
```
Comment (tylerbutler) [inline packages/drivers/odsp-driver-definitions/package.json:98]: Noting for follow up - find the commit which would have added the "broken" entries.

File: packages/drivers/file-driver/package.json [hunk 3]
```diff
@@ -80,7 +80,7 @@
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/build-tools": "^0.40.0",
 		"@fluidframework/eslint-config-fluid": "^5.3.0",
-		"@fluidframework/file-driver-previous": "npm:@fluidframework/file-driver@2.0.0-rc.5.0.0",
+		"@fluidframework/file-driver-previous": "npm:@fluidframework/file-driver@2.0.0",
```
Comment (tylerbutler) [inline packages/drivers/file-driver/package.json:83]: Noting for follow up - find the commit which would have added the "broken" entries.

File: packages/service-clients/azure-client/package.json [hunk 4]
```diff
@@ -109,7 +109,7 @@
 		"@biomejs/biome": "^1.7.3",
 		"@fluid-tools/build-cli": "^0.40.0",
 		"@fluidframework/aqueduct": "workspace:~",
-		"@fluidframework/azure-client-previous": "npm:@fluidframework/azure-client@2.0.0-rc.5.0.0",
+		"@fluidframework/azure-client-previous": "npm:@fluidframework/azure-client@2.0.0",
```
Comment (tylerbutler) [inline packages/service-clients/azure-client/package.json:112]: Noting for follow up - find the commit which would have added the "broken" entries.

**Review by tylerbutler — APPROVED** [review]
Looks good. I made notes on the four packages which still had overrides. I want to verify that if we'd bumped typetests right after 2.0 that the configs would be 100% clean. Not blocking for this PR.

**Comment by CraigMacomber** [comment]
> Looks good. I made notes on the four packages which still had overrides. I want to verify that if we'd bumped type tests right after 2.0 that the configs would be 100% clean. Not blocking for this PR.

I did look at the git blame for the ignores. None were too old or looked too suspicious, so I was planning to do a more robust check when we do 2.1, but if you want to properly validate this, that would be good.

**Pull request merged**